source/diagrams/manage-diagrams/clone-diagrams.txt

```diff
@@ -0,0 +1,13 @@
+.. _clone_diagrams:
+
+==============
+Clone Diagrams
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+MySQL.
```

Comment:
*Nit* Could you update this to the name of the pages for all new pages?

Reply:
Done!